ios/BranchLinkProperties+RNBranch.m

```diff
@@ -0,0 +1,44 @@
+//
+//  BranchLinkProperties+RNBranch.m
+//  RNBranch
+//
+//  Created by Jimmy Dee on 1/26/17.
+//  Copyright © 2017 Dispatcher. All rights reserved.
+//
+
+#import "BranchLinkProperties+RNBranch.h"
+#import "NSObject+RNBranch.h"
+#import "RNBranchProperty.h"
+
+@implementation BranchLinkProperties(RNBranch)
+
++ (NSDictionary<NSString *,RNBranchProperty *> *)supportedProperties
+{
+    static NSDictionary<NSString *, RNBranchProperty *> *_linkProperties;
+    static dispatch_once_t once = 0;
+    dispatch_once(&once, ^{
+        _linkProperties =
+        @{
+          @"alias": [RNBranchProperty propertyWithSetterSelector:@selector(setAlias:) type:NSString.class],
+          @"campaign": [RNBranchProperty propertyWithSetterSelector:@selector(setCampaign:) type:NSString.class],
+          @"channel": [RNBranchProperty propertyWithSetterSelector:@selector(setChannel:) type:NSString.class],
+          // @"duration": [RNBranchProperty propertyWithSetterSelector:@selector(setMatchDuration:) type:NSNumber.class], // deprecated
```

Comment:
I don't know why duration is deprecated (or if it really is), but it says so [here](https://dev.branch.io/getting-started/configuring-links/guide/#other-parameters). It's easy to support it if that's wrong.